src/server/db_setup.ts

```diff
@@ -257,3 +257,25 @@ export async function insertGameResults(
         return { status: 200, results: [] };
     }
 }
+
+//Function for adding a log to the log database
+export async function insertLog(log: string): Promise<GameResultsFormResult> {
+    //TODO: implement
+
+    //Parse the data
+    log = trimLog(log);
+    console.log(log);
+
+    //Placeholder return statement
+    return { status: 200, results: [] };
+}
+
+//Helper function for trimming a log
+function trimLog(log: string): string {
+    //TODO: implement, may need more processing
+
+    //Removes < > and any characters between them, will not work with nested tags but should be fine with the HTML elements in the logs
```

Comment:
The other piece to remove would be `</tag>`. It might also be useful to use HTML tags as a token indicator / break it into a list.
Consider:
```<span style=\"color:yellow\">g</span> draws <span><span style=\"color:white\">5 cards</span></span>.```
If we simple remove tags, we get `g draws 5 cards.` followed by more "sentences". But these tags do give us a logical organization, and another span like this following is another "sentence", instead of just looking for periods to indicate statement/sentence ends. Maybe. If you want to look at a real HTML parser, start with https://www.npmjs.com/package/node-html-parser